src/isIsogram.test.js

```diff
@@ -6,4 +6,40 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return a boolean type`, () => {
+    expect(typeof isIsogram('sport')).toBe('boolean');
+  });
+
+  it(`should return a true if word === ''`, () => {
+    expect(isIsogram('')).toBeTruthy();
+  });
+
+  it(`should return a true if word.length === 1`, () => {
+    expect(isIsogram('A')).toBeTruthy();
+  });
+
+  it(`should not be case sensitive`, () => {
+    expect(isIsogram('Abba')).toBeFalsy();
+  });
+
+  it(`should return false if word === 'aA'`, () => {
```

Comment:
It would be better to write 
should return false when input is 'aA'

Use this rule for your following tests